snap-css-core/src/css/css.ts

```diff
@@ -1,9 +1,11 @@
 import RuleSet from './rule-set'
 import * as postcss from 'postcss'
+const expand = require('postcss-shorthand-expand')
```

Comment:
postcss-shorthand-expand currently only supports 12 shorthand properties. On the other hand, https://www.npmjs.com/package/css-shorthand-expanders supports around 32 shorthand properties. Check it out